pkg/quarterdeck/db/models/orgs.go

```diff
@@ -20,10 +20,11 @@ import (
 // used for API serialization.
 type Organization struct {
 	Base
-	ID       ulid.ULID
-	Name     string
-	Domain   string
-	projects int
+	ID        ulid.ULID
+	Name      string
+	Domain    string
+	projects  int
+	lastLogin sql.NullString
```

Comment:
~Is the idea that this is the last login for any user?~

Ok, I see it's the last login for the user querying the org list.